.github/workflows/node.js.yml

```
@@ -5,26 +5,73 @@ name: Node.js CI
-    branches: [ "main" ]
+    branches: ["main"]
   pull_request:
-    branches: [ "main" ]
-
+    branches: ["main"]
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
+    container:
```

actionlint (1.7.7)

string should not be empty

(syntax-check)